x-pack/plugins/alerting/public/pages/maintenance_windows/components/recurring_schedule_form/custom_recurring_schedule.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import moment from 'moment';
+import { css } from '@emotion/react';
+import { getUseField, useFormData } from '@kbn/es-ui-shared-plugin/static/forms/hook_form_lib';
+import { Field } from '@kbn/es-ui-shared-plugin/static/forms/components';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { CREATE_FORM_CUSTOM_FREQUENCY, Frequency, WEEKDAY_OPTIONS } from '../../constants';
+import * as i18n from '../../translations';
+import { ButtonGroupField } from '../fields/button_group_field';
+import { getInitialByWeekday } from '../../helpers/get_initial_by_weekday';
+import { getWeekdayInfo } from '../../helpers/get_weekday_info';
+
+const UseField = getUseField({ component: Field });
+
+const styles = {
```

Comment:
So I checked this out, the space is created by the labelWrapper, I noticed in `schema.tsx`, the custom frequency field is 

```
    customFrequency: {
      type: FIELD_TYPES.SELECT,
      label: '',
      defaultValue: Frequency.WEEKLY,
    },
```

but if you set the label to `' '` it fixes the space (the space string adds this empty `labelWrapper`) So we can probably remove this style. I guess the question is shouldn't all of the labels be empty if there's no labels?